### PR TITLE
Add Lots of Loading Spinners

### PR DESCRIPTION
**Pull request by dhaselhan**
Spinners added to:
* Applicant Info (NOI+APP)
* App Prep Parcels (NOI+APP)
* View App in Portal (NOI+APP)
* Public View App in Portal (NOI+APP+SRW)

### CODE REVIEW

File: alcs-frontend/src/app/features/notice-of-intent/applicant-info/notice-of-intent-details/notice-of-intent-details.component.ts [hunk 1]
```diff
@@ -56,11 +56,12 @@ export class NoticeOfIntentDetailsComponent implements OnInit, OnChanges, OnDest
...
       (document) =>
         document.type &&
-        [DOCUMENT_TYPE.PHOTOGRAPH, DOCUMENT_TYPE.OTHER, DOCUMENT_TYPE.PROFESSIONAL_REPORT].includes(document.type.code)
+        [DOCUMENT_TYPE.PHOTOGRAPH, DOCUMENT_TYPE.OTHER, DOCUMENT_TYPE.PROFESSIONAL_REPORT].includes(document.type.code),
     );
     this.authorizationLetters = documents.filter(
-      (document) => document.type?.code === DOCUMENT_TYPE.AUTHORIZATION_LETTER
+      (document) => document.type?.code === DOCUMENT_TYPE.AUTHORIZATION_LETTER,
     );
+    debugger;
```
Comment (mhuseinov) [inline alcs-frontend/src/app/features/notice-of-intent/applicant-info/notice-of-intent-details/notice-of-intent-details.component.ts:64]: looks like a debug leftover

**Review by mhuseinov — APPROVED** [review]
👍

File: alcs-frontend/src/app/features/notice-of-intent/applicant-info/notice-of-intent-details/parcel/parcel.component.ts [hunk 2]
```diff
@@ -43,8 +43,11 @@ export class ParcelComponent implements OnInit, OnChanges, OnDestroy, AfterConte
 
   async onOpenFile(uuid: string) {
     const file = this.files.find((file) => file.uuid === uuid);
+    debugger;
```
[the remark this answers is not in the document]
Comment (dhaselhan) [reply alcs-frontend/src/app/features/notice-of-intent/applicant-info/notice-of-intent-details/parcel/parcel.component.ts]: removed

File: alcs-frontend/src/app/features/notice-of-intent/applicant-info/notice-of-intent-details/notice-of-intent-details.component.ts (hunk 1 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/notice-of-intent/applicant-info/notice-of-intent-details/notice-of-intent-details.component.ts]: removed

Comment (trslater) [inline alcs-frontend/src/app/features/notice-of-intent/applicant-info/notice-of-intent-details/notice-of-intent-details.component.ts]: Here as well.

**Review by trslater — APPROVED** [review]
LGTM 👍

**Pull request merged**